src/configure.js

```diff
@@ -66,7 +66,7 @@ export default (config) => {
         ...page,
         id: ++pageId,
         // Currently, catalog can't be nested inside other page routes, it messes up <Link> matching. Use `basePath`
-        path: parsePath(page.path || page.name, {basePath}).pathname,
+        path: page.pages ? null : parsePath(page.path || page.name, {basePath}).pathname,
```

Comment:
I like this behavior: if `pages` are defined it really doesn't make sense to have a `path`, so explicitely setting it to `null` (even if it were available) is a good choice.